plansys2_tools/src/plansys2_logger/LoggerNode.cpp

```diff
@@ -15,6 +15,7 @@
 
 #include <string>
 #include <memory>
+#include <iomanip>
```

Comment:
This include is not necessary

Reply:
I believe it is on foxy for some reason (default C++ version?), but @roveri-marco can confirm/deny

Reply:
I needed otherwise it complains for setprecision being undefined. 
It is not a matter of C++ version, it might be that in galactic it has been added somewhere else and thus it is not needed, but from a clean C++ perspective if you need to use the setprecision you need that include. Thus, I would push to have it.

Reply:
ok